src/helpers/offlineSyncModal.js

```diff
@@ -63,11 +65,16 @@ function offlineSyncModalFactory(proxy) {
     //creates the waiting modal dialog
     const waitingDialog = waitingDialogFactory(waitingConfig)
         .on('render', () => {
+            delaySec = secondaryButtonWait;
             $secondaryButton = $('div.preview-modal-feedback.modal').find('button[data-control="secondary"]');
```

Comment:
didn't notice it, but this is also an issue. The selector should target the proper dialog. What will happen if another dialog is open at the same time?